perception/object_recognition/detection/tensorrt_yolo3/src/tensorrt_yolo3_main.cpp

```diff
@@ -17,10 +17,13 @@
 
 int main(int argc, char ** argv)
 {
-  ros::init(argc, argv, "tensorrt_yolo3");
-  TensorrtYoloROS app;
-  app.createROSPubSub();
-  ros::spin();
+  rclcpp::init(argc, argv);
+
+  const auto node = std::make_shared<TensorrtYoloROS>();
+  node->createROSPubSub();
```

Comment:
Even if it's not part of the porting, I'd strongly recommend moving this code to the constructor so that all main files look the same (for the eventual migration to components).

Reply:
moved in cdc5e7e990dfdecc23c8d9b07b0b37413a41fa41